composer/datasets/ade20k.py

```diff
@@ -54,17 +54,53 @@ class RandomCropPair(torch.nn.Module):
         crop_size (Tuple[int, int]): the size (height x width) of the crop.
     """
 
-    def __init__(self, crop_size: Tuple[int, int]):
+    def __init__(self, crop_size: Tuple[int, int], class_max_percent: float = 1.0, n_retry: int = 0):
```

Comment:
add these new args to docstring.

Reply:
Oops, should have done that before review request :(